src/assets/scss/component/_custom-form-styles.scss

```diff
@@ -58,502 +55,494 @@ $formContainerBorderColor: #ccc;
-  background-color: $confirmationBannerBackground
-}
-.errorbox-bad {
-  border: 2px solid $error;
-  background-color: $errorBoxBadBorderColor;
-  padding: 2px;
-}
-.error-message {
-  display: none;
-}
-
-.errorheader {
-  color: $error
-}
-input#emailReceipt {
-  margin-left: 0;
-  margin-top: 0;
-  vertical-align: middle;
-}
-.no-chrome .powered-by-logo, .no-chrome .ss-legal, .no-chrome .ss-edit-link {
-  display: none
-}
-.ss-prepopulate-input, .ss-submit-url-input {
-  width: 100%
+.google-form {
+  html .goog-inline-block,
+  *:first-child+html .goog-inline-block {
+    display: inline;
+  }
+  body {
+    color: $baseColor;
+    font-family: $bodyFontFamily;
+    font-weight: 400;
+    margin: 0;
+    line-height: $bodyLineHeight;
+    padding: 0 1em;
+    background-color: white;
+    font-weight: normal;
+  }
+  input,
+  textarea {
+    outline: none;
+  }
+  .container {
+    margin: 0 auto;
+    width: 960px;
+  }
+  .ss-logo-link {
+    display: block;
+  }
+  .ss-logo {
+    border: none;
+    height: 23px;
+    width: 105px;
+  }
+  .g {
+    color: $baseColor;
+  }
+  .ss-form-desc {
+    font: inherit;
+    width: 99%;
+    margin: 0;
+  }
+  .ss-powered-by {
+    display: block;
+    clear: left;
+    color: $baseColor;
+    margin: 1em 0.2em 0.2em;
+  }
+  .ss-powered-by a:link,
+  .ss-powered-by a:visited {
+    color: $baseColor;
+  }
+  .ss-required-asterisk,
+  .ss-required {
+    color: $error;
+  }
+  .ss-no-ignore-whitespace {
+    white-space: pre-wrap;
+    word-wrap: break-word;
+  }
+  .ss-confirmation-header {
+    font-size: 140%;
+    padding-bottom: 1em;
+  }
+  .ss-abuse-header {
+    background: #fff68f;
+    line-height: 1.3em;
+    margin: 0 auto 1em;
+    padding: 0.5em;
+    width: 97%;
+  }
+  .ss-navigate {
+    clear: both;
+  }
+  .form-error-arg {
+    font-weight: bold;
+  }
+  h1 {
+    font-size: 1.8em;
+    padding: 0;
+  }
+  hr {
+    border: 0;
+    height: 1px;
+    background-color: $hrBackground;
+  }
+  table {
+    border-collapse: collapse;
+  }
+  textarea {
+    font: normal 13px arial,sans-serif;
+  }
+  br {
+    display: block!important;
+  }
+  .aria-only-help {
+    font-size: 0;
+    left: -9999px;
+    position: absolute;
+  }
+  .ss-printable-hint {
+    display: none;
+  }
+  .goog-date-picker,
+  .goog-date-picker th,
+  .goog-date-picker td {
+    font: 13px Arial,sans-serif;
+  }
+  .goog-date-picker {
+    -moz-user-focus: normal;
+    -moz-user-select: none;
+    position: relative;
+    border: 1px solid $datePickerBorder;
+    float: left;
+    padding: 2px;
+    color: $datePickerColor;
+    background: $datePickerBackground;
+    cursor: default;
+  }
+  .goog-date-picker th {
+    text-align: center;
+  }
+  .goog-date-picker td {
+    text-align: center;
+    vertical-align: middle;
+    padding: 1px 3px;
+  }
+  .goog-date-picker-menu {
+    position: absolute;
+    background: $datePickerMenuBackground;
+    border: 1px solid gray;
```

Comment:
Color `gray` should be written in hexadecimal form as `#808080`<br>Color literals like `gray` should only be used in variable declarations; they should be referred to via variable everywhere else.